nestkernel/nest_names.h

```diff
@@ -31,613 +31,503 @@ namespace nest
 
 /**
  * This namespace contains global Name objects. These can be used in
- * Node::get_status and Node::set_status to make data exchange
- * more efficient and consistent. Creating a Name from a std::string
- * is in O(log n), for n the number of Names already created. Using
- * predefined names should make data exchange much more efficient.
+ * Node::get_status and Node::set_status to make data exchange more
+ * efficient and consistent. Creating a Name from a std::string is in
+ * O(log n), for n the number of Names already created. Using
+ * predefined names makes data exchange much more efficient as it
+ * uses integer comparisons instead of string comparisons internally.
  */
```

Comment:
Please extend this comment to explain the rules of ordering in this file.

Reply:
I've extended the comment in 50b99d1ed62.